### ISSUE TITLE
Add GS1 DataMatrix

### ISSUE THREAD

**Issue by glockmane** [Siox0911/OBIforExcel#1]
Should be a very minor change but GS1 DataMatrix is often used.

https://www.bluhmsysteme.com/blog/datamatrix-und-gs1-datamatrix-der-kleine-feine-unterschied/

Thank you for considering!

**Comment by Siox0911** [comment]
Yes I know the GS1 Data Matrix. It is standard in many logistics areas. But it is not so easy to integrate this code. My solution uses the open source lib DataMatrix.Net for building the Data Matrix codes. I don't even know if that's supported.
